source/braille.py

```diff
@@ -2009,6 +2011,37 @@ def formatCellsForLog(cells: List[int]) -> str:
 """
 
 
+# The list containing the regions that will be shown in braille when the speak function is called
+# and the braille mode is set to speech output
+_regions: list[TextRegion] = []
+
+
+def _showSpeechInBraille(speechSequence: "SpeechSequence"):
+	if config.conf["braille"]["mode"] == BrailleMode.FOLLOW_CURSORS.value:
+		return
+	regionsText = "".join([i.rawText for i in _regions])
+	if len(regionsText) > 100000:
+		return
+	text = " ".join([x for x in speechSequence if isinstance(x, str)])
+	currentRegions = False
+	if _regions:
+		text = " " + text
+		currentRegions = True
+	region = TextRegion(text)
+	region.update()
+	_regions.append(region)
+	handler.mainBuffer.regions = _regions.copy()

```

Comment:
Given that this function is pretty strictly tied to the braille handler, I wonder whether it should be a method on the handler itself.